terraform/03_gke_cluster.tf

```diff
@@ -67,8 +67,6 @@ resource "google_container_cluster" "gke" {
   # interesting things.
   node_pool {
     node_config {
-      machine_type = "n1-standard-2"
```

Comment:
Don't we need to specify the machine_type anymore?

Reply:
@Daniel-Sanche - was there any change related to this?

Reply:
It didn't modify this line.

Reply:
We do need this line. Please make sure you keep changes from upstream